SecurityPkg/Include/Library/Tcg2InitEventLib.h

```diff
@@ -0,0 +1,57 @@
+/** @file
+  This describes the interface that should be published by instances of the
+  Tcg2InitEventLib. This library can be used to initialize events for logging
+
+Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef TCG_2_INIT_EVENT_LIB_H_
+#define TCG_2_INIT_EVENT_LIB_H_
+
+#include <IndustryStandard/UefiTcgPlatform.h>
+
+/**
+  This function initialize TCG_PCR_EVENT2_HDR for EV_NO_ACTION Event Type other than EFI Specification ID event
+  The behavior is defined by TCG PC Client PFP Spec. Section 9.3.4 EV_NO_ACTION Event Types

```

Comment:
This is a little much to take in all at once. Does this still capture what is being stated?

```suggestion
  This function initializes the TCG_PCR_EVENT2_HDR for EV_NO_ACTION events.
  
  Note: The EFI Specification ID event is not supported.

  The behavior is defined by TCG PC Client PFP Spec. Section 9.3.4 EV_NO_ACTION Event Types.
```